cli/main.py

```diff
@@ -678,217 +681,6 @@ def stake(config,
-                value = random.randint(min_stake, balance)                            # stake a random amount...
-                min_locktime, max_locktime = MIN_LOCKED_PERIODS, MAX_MINTING_PERIODS  # ...for a random lock duration
-                periods = random.randint(min_locktime, max_locktime)
-                process_params.extend('--checksum-address {}'.format(sim_address).split())
-                process_params.extend('--stake-amount {} --stake-periods {}'.format(value, periods).split())
-
-            # Spawn
-            click.echo("Spawning node #{}".format(counter+1))
-            processProtocol = UrsulaCommandProtocol(command=process_params, checksum_address=sim_address)
-            cli_exec = os.path.join(BASE_DIR, 'cli', 'main.py')
-            ursula_process = reactor.spawnProcess(processProtocol=processProtocol,
-                                                  executable=cli_exec,
-                                                  args=process_params,
-                                                  env=os.environ)
-
-            config.sim_processes.append(ursula_process)
-
-            #
-            # post-spawnProcess
-            #
-
-            # Start with some basic status data, then build on it
-
-            rest_uri = "https://{}:{}".format('localhost', sim_port_number)
-
-            sim_data = "prepared simulated Ursula | ReST {}".format(rest_uri)
-            rest_uri = "{host}:{port}".format(host='localhost', port=str(sim_port_number))
-            sim_data.format(rest_uri)
-
-            if not config.federated_only:
-                sim_data += '| ETH address {}'.format(sim_address)
-
-            click.echo(sim_data)
-            counter += 1
-
-        click.echo("Starting the reactor")
-        click.confirm("Start the reactor?", abort=True)
-        try:
-            reactor.run()
-        finally:
-            if not config.federated_only:
-                click.echo("Removing simulation registry")
-                os.remove(DEFAULT_SIMULATION_REGISTRY_FILEPATH)
-            click.echo("Stopping simulated Ursula processes")
-            for process in config.sim_processes:
-                os.kill(process.pid, 9)
-                click.echo("Killed {}".format(process))
-            click.echo("Simulation Stopped")
-
-    elif action == 'stop':
-        # Kill the simulated ursulas
-        for process in config.ursula_processes:
-            process.transport.signalProcess('KILL')
-
-    elif action == 'status':
-        if not config.simulation_running:
-            status_message = "Simulation not running."
-        else:
-            ursula_processes = len(config.ursula_processes)
-            status_message = """
-            
-            | Node Swarm Simulation Status |
-            
-            Simulation processes .............. {}
-            
-            """.format(ursula_processes)
-        click.echo(status_message)
-
-    else:
-        raise click.BadArgumentUsage
-
-
```

Comment:
Phew!  That's a breath of fresh air right there.  A goddam strawberry patch of redlines in here.